### PR TITLE
Angel prats

### PR DESCRIPTION
**Pull request by angelprats**

### CODE REVIEW

File: Answers.md [hunk 1]
```diff
@@ -1,6 +1,44 @@
...
+
+
+
 1.  Describe `redux-thunk`, what does it allow us to do? How does it change our `action-creators`?
+redux thunk is a function returned by another function, used in action creators for asynchronous operations.
+
+
+
 1.  Which `react-redux` method links up our `components` with our `redux store`?
+connect.
```
Comment (RaineBeauDubs) [inline Answers.md:44]: Thank you for your answers!

File: smurfs/src/components/Smurf.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Smurf = props => {
+  return <li>{props.smurf.name}</li>
+}
+
+export default Smurf;
```
Comment (RaineBeauDubs) [inline smurfs/src/components/Smurf.js:7]: I think this whole single smurf component isn't actually doing anything, you could just take it out if you wanted to make your code more DRY.

File: smurfs/src/components/SmurfList.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    this.props.getSmurf();
+  }
+  render() {
+    return (
+      <div>
+        {this.props.smurfs.map(smurf =>
+          <div>
+            <h1>{smurf.name}</h1>
+            <h2>{smurf.age}</h2>
+            <h2>{smurf.height}</h2>
```
Comment (RaineBeauDubs) [inline smurfs/src/components/SmurfList.js:20]: See, this _here_ is where your single smurf information is being rendered.

**Review by RaineBeauDubs — APPROVED** [review]
I left a couple pieces of feedback for you, but overall, great job, Angel!

**Comment by angelprats** [comment]
Done!. You we right! I removed it. Ty.

**Pull request merged**